### ISSUE TITLE
View Helper Implementation is set before the initializer gets called 

### ISSUE THREAD

**Issue by bogdan-dumitru** [reactjs/react-rails#550]
- [x] My app doesn't crash, but I'm getting unexpected behavior. So, I've described the unexpected behavior and suggested a new behavior.

-------

I'm trying to use a custom view helper implementation class that I've defined and set in my initializer file like so:
```ruby
  config.react.view_helper_implementation = CustomViewHelper
```
But the default ``ComponentMount`` ends up being used.
I added some breakpoints in the init process to try to figure things out and as far as I can tell the [initializer block for "react_rails.setup_view_helpers"](https://github.com/reactjs/react-rails/blob/master/lib/react/rails/railtie.rb#L28-L45) gets executed _before_ my initializer sets the config class. I added breakpoints and that's the order I saw, and it explains the behaviour, but I'm unsure if dev environment affects this somehow.

Either I'm missing something or these init blocks should run after the app gets initialised. The code there depends on other config variables as well (that end up being the defaults) and there are other blocks which would fall under the same issue. Is it a general problem or am I missing something?

**Comment by rmosolgo** [comment]
I'm _guessing_ it's a general problem, but I haven't noticed it because I only use the default ones! 

Maybe that initialization should be moved to the `after_initialize` block at the bottom of the file? 

(By the way, for a workaround, could you assign the implementation directly? 

```ruby 
 React::Rails::ViewHelper.helper_implementation_class = CustomViewHelper
```

?)

**Comment by bogdan-dumitru** [comment]
Yeah that's what I ended up doing, assigning the implementation directly, and it works.

**Comment by bogdan-dumitru** [comment]
@rmosolgo Actually I think cleanest solution would be to add an ``:after => :load_config_initializers`` to all the initializer blocks defined there, to ensure they are called after we've set the configs in the initializer.
I'm happy to give that a go and open a pull request if it sounds good.

**Comment by rmosolgo** [comment]
Yes, that would be great! I didn't know about `:load_config_initializers`, good to know!

**Comment by danott** [comment]
@rmosolgo I believe this can be closed now that #551 has been merged?

**Comment by rmosolgo** [comment]
yes, thanks!
